estate/models/estate_property.py

```diff
@@ -0,0 +1,84 @@
+from dateutil.relativedelta import relativedelta
+from odoo import fields, models,api,exceptions
+from odoo.tools.float_utils import float_compare, float_is_zero;
+from odoo.exceptions import ValidationError;
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Property"
+    _order = "id desc"
+
+    _positif_expected_price = models.Constraint("CHECK (expected_price > 0)","A price can't be negatif");
+    _positif_selling_price = models.Constraint("CHECK (selling_price > 0)","A price can't be negatif");
+
+    state = fields.Selection(selection = [("New","New"), ("Offer_Received","Offer Received") ,("Offer_Accepted","Offer Accepted"), ("Sold","Sold"), ("Cancelled","Cancelled")])
```

Comment:
for the selection fields I prefer to write them that way to make the choices clear while we look at the field. and for the values the first part of each tuple is the technical value for the field which should be written like a var name in snake case 🐍 
```suggestion
    state = fields.Selection([
        ("new","New"),
        ("offer_received","Offer Received"),
        ("offer_accepted","Offer Accepted"),
        ("sold","Sold"),
        ("cancelled","Cancelled")
    ], default="new")
```